rpc/handle_get_net_totals.go

```diff
@@ -1,16 +1,16 @@
 package rpc
 
 import (
-	"github.com/kaspanet/kaspad/rpcmodel"
+	"github.com/kaspanet/kaspad/rpc/model"
 	"time"
 )
 
 // handleGetNetTotals implements the getNetTotals command.
 func handleGetNetTotals(s *Server, cmd interface{}, closeChan <-chan struct{}) (interface{}, error) {
-	totalBytesRecv, totalBytesSent := s.cfg.ConnMgr.NetTotals()
-	reply := &rpcmodel.GetNetTotalsResult{
-		TotalBytesRecv: totalBytesRecv,
-		TotalBytesSent: totalBytesSent,
+	// TODO(libp2p): fill this up with real values
```

Comment:
Please create ticket if you haven't yet.

Reply:
Created.